project-1/my-project-1202.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Not a good practice to commit credentials, please use a .env and add to .gitignore